From 16fd65521a347a8a2d2b63399775fb60b9c97d56 Mon Sep 17 00:00:00 2001
From: Jakub Jelen <jjelen@redhat.com>
Date: Thu, 22 Oct 2026 13:10:14 +0200
Subject: [PATCH] Verify source signatures with openpgpverify

Replace the GnuPG-based %gpgverify with %openpgpverify, which uses
Sequoia sqv and takes the same options, as part of the Fedora 45 Change
https://fedoraproject.org/wiki/Changes/Sequoia_openpgpverify

Refresh dickey@invisible-island.net-rsa3072.asc from keyservers/WKD: same certificate(s), with self-signatures our copy was missing.
Release bumped with a changelog entry.

Assisted-by: Claude Opus 5.5
---
 dialog.spec                             |  10 +-
 dickey@invisible-island.net-rsa3072.asc | 225 ++++++++++++++++++++----
 2 files changed, 200 insertions(+), 35 deletions(-)

diff --git a/dialog.spec b/dialog.spec
index 368c633..9a47f96 100644
--- a/dialog.spec
+++ b/dialog.spec
@@ -2,13 +2,14 @@ Summary: A utility for creating TTY dialog boxes
 Name: dialog
 %global dialogsubversion 20250116
 Version: 1.3
-Release: 58.%{dialogsubversion}%{?dist}
+Release: 59.%{dialogsubversion}%{?dist}
 License: LGPL-2.1-only
 URL: https://invisible-island.net/dialog/dialog.html
 Source0: https://invisible-mirror.net/archives/dialog/dialog-%{version}-%{dialogsubversion}.tgz
 Source1: https://invisible-mirror.net/archives/dialog/dialog-%{version}-%{dialogsubversion}.tgz.asc
 Source2: https://invisible-island.net/public/dickey@invisible-island.net-rsa3072.asc
-BuildRequires: ncurses-devel gcc gettext findutils libtool gnupg2
+BuildRequires: ncurses-devel gcc gettext findutils libtool
+BuildRequires: openpgpverify
 BuildRequires: make
 Patch2: dialog-multilib.patch
 Patch3: dialog-libs.patch
@@ -33,7 +34,7 @@ contains the files needed for developing applications, which use the
 dialog library.
 
 %prep
-%{gpgverify} --keyring=%{SOURCE2} --signature=%{SOURCE1} --data=%{SOURCE0}
+%{openpgpverify} --keyring=%{SOURCE2} --signature=%{SOURCE1} --data=%{SOURCE0}
 %setup -q -n dialog-%{version}-%{dialogsubversion}
 %patch -P2 -p1 -b .multilib
 %patch -P3 -p1 -b .libs
@@ -80,6 +81,9 @@ rm -f $RPM_BUILD_ROOT%{_libdir}/libdialog.{,l}a
 %{_mandir}/man3/dialog.*
 
 %changelog
+* Mon Oct 12 2026 Jakub Jelen <jjelen@redhat.com> - 1.3-59.20250116
+- Verify upstream source signatures with openpgpverify
+
 * Wed Jul 15 2026 Fedora Release Engineering <releng@fedoraproject.org> - 1.3-58.20250116
 - Rebuilt for https://fedoraproject.org/wiki/Fedora_45_Mass_Rebuild
 
diff --git a/dickey@invisible-island.net-rsa3072.asc b/dickey@invisible-island.net-rsa3072.asc
index 33a8ce3..cafcef9 100644
--- a/dickey@invisible-island.net-rsa3072.asc
+++ b/dickey@invisible-island.net-rsa3072.asc
@@ -1,6 +1,6 @@
 -----BEGIN PGP PUBLIC KEY BLOCK-----
 
-mQGNBGEEhmgBDAD3TTylJvq42rjYW1ISL2bXcNDvvqOyzhacLayhgHwe90yHPRtH
+xsDNBGEEhmgBDAD3TTylJvq42rjYW1ISL2bXcNDvvqOyzhacLayhgHwe90yHPRtH
 6v4kyOnltv3txWmbHpqSPFKOj+hi6SYO5o0iIc81p0BCtqu6za8WoGqfvyZ/N5V/
 dNkKcU3dRVBgub62ZlRMagWWBvHY8Hu6FqDHwlaQqJtckUIfbVbUDteMYOhIZ6Fb
 rdT4d4n1TMIN6pqgNcs6yHSrqfZUkgWkRMVtxf+nu6X0H1ntbn/9X0UqOG/vcoMD
@@ -8,35 +8,196 @@ j5QQK8s1xItCovjy69UNRDV+bvtb9YOXTkv1tDgqPCBGidOr9oznN91S5hbUjoUu
 rZNwlZsQZ7GGz5MXt2I4aL6sPyvKSj7IdVa3R2g+z8QLxvQWCN6V2FOcrziZ/EK+
 Rpnaw3C7+siz+lWGxVo1CJBks+V0lviA9Sd9MLJBJoxgJSao1YLkBSE9UXTBx1Va
 RU1buTg/0+2BrxEdX+9qIb4Qp/kaa33+Jw1IDauO6TQALilw2D1idkLZwlzMcfH0
-Mvlvd6DVWkVZ29EAEQEAAbRFVGhvbWFzIEUuIERpY2tleSAoc2VsZi1zaWduZWQg
-dy9vIFNIQTEpIDxkaWNrZXlAaW52aXNpYmxlLWlzbGFuZC5uZXQ+iQHUBBMBCgA+
-FiEEGYgtkt2kxADCLA1WzCr0RyFnvgMFAmEEhmgCGwMFCQPCZwAFCwkIBwIGFQoJ
-CAsCBBYCAwECHgECF4AACgkQzCr0RyFnvgNwgQv/cwwRUoDCw9C6OjblqEa2eA0O
-TwkKqT5QcUUtXsMTNHZ5RZ8BHbJjiODTC4uKt7IaxkcYH72MLbMQLXAybJJhJ9Gs
-JOvtu7OXS+tKhRjCYrdooFyupjuhq7kzY9Q7AsNPYyfGjLzxkJx4DQYe7UzzIsMn
-FkUBP4nBl4VwzfFyZrKT3WO7+1RWyuNKVwX39vF7JZH8C7j/FE8iFxCHJvfzqs5S
-MSw4EkvJFRMdqljNkypMEqW4jh3T1RGU3/BXfwgJE3MJ+F6kdgXGe9UecKx/Arb1
-n7JxCfIe4cq/J2jiWnhGXqE/Jay8n4voC0LHhrLIjoQQkFMDpqp9rc+bqEjVm/CA
-Wuv+f0F5Ppf4ctGrat+Q0fOTYo80jgFPHw3krJLgAxP9EaIPfX31zFnwyEdwfYy+
-qaLnmyNAjUR/KAilW9tagXLPXI9TJeJxV5dbT5XbDY4cx5ksjtuA8fcRyKtcmi4U
-jya6rrcrCgr5Ooix8UHUEzDTNZWlMvmG6GMKsr3PuQGNBGEEhmgBDAC8liudPkt0
-jbBeUvOvVun2rhPSf7Ge4i4xNO2LjM3o6hy2xsqKqi5QhUl9vasf4ObcYh8rh2Wx
-wDWJcLhcQSTRtRAufqjdVtemH9hncUsn28HRuw/E7syYQbRBdXVE1BTWFu6byheK
-crZ7uV8tpB5FTZSfacLYVUc1GZSy3Q1J04aVqWZWre9fkN9ELVJp9sM96s/20Z60
-xrCyQSW9d+FM/NzB0C7O77QrrTEcotcXGi0GbRD6FCz518lcwiTFU+YvhpD9QSek
-UlujF2keHFMdajH2aHuIzyuu2k8bbN80fG7s/G8C+hNwIlcR05a7f0o9loKUwq7E
-67Fs4P6ZSXmoYU5wGBzsMoM8hM7UsvIDX28KRcz9n753NsIX2HtO26sJu3Z9kBj1
-SFdUTV20sxbKmbF6I0etVpc7x4tc1bBPXEFKeBZph+Zu3mzxcvtlZYraOuqSZFr0
-4Mx+4njIKeHklwr/GW5oaZG1JIPH0HddzPROUpwYXElCJRiEq1ObNesAEQEAAYkB
-vAQYAQoAJhYhBBmILZLdpMQAwiwNVswq9EchZ74DBQJhBIZoAhsMBQkDwmcAAAoJ
-EMwq9EchZ74DamgL/3ipBSBQgvLk/4A7y1mC1fntoaZwwcGJxgXsqCId8jo4jBFC
-yg72nLrDEHYH9AWy1tS4xXdFw0gEAnb2pm87t+GkKoGdeyH2sgFNfMT106OgQRpk
-Vmg7NoVc+qPJcmJCOU1jqSx1PvmJPM/mhTftf+KLljP8a6XhMy1IuhqonOA1HTtf
-ebo+J/dZmMjCGQKoP2ib8WenI+JAdtuEIw5BUCEhosRnGuMuBbrZwq4+usdZQy70
-9l0TeUQLDvKJJrycpy93HrB38Z299QmwlxAgUv3gPjiqf48JC8SqlOnbRVcZXNlb
-0EnypnBxSjo+myC1sysl4sa23XN64+a/yMmDvB2VQykB2bKS1B1aVbGxMYtSctV+
-+7kC3BYuFXkTkXWybbzvqPp9Nci2cugeak1qwJRZq7f0s5oJZobaSqkMIwXmY9Gj
-snvWC9WYkWAfYL92TPghI8LOawz6UW2TKePrZlTRlApeOwc5M3K2ep1ScW8aJO3X
-P46cd3aSov0rfNS/zg==
-=cr15
+Mvlvd6DVWkVZ29EAEQEAAc1FVGhvbWFzIEUuIERpY2tleSAoc2VsZi1zaWduZWQg
+dy9vIFNIQTEpIDxkaWNrZXlAaW52aXNpYmxlLWlzbGFuZC5uZXQ+wsEUBBMBCgA+
+AhsDBQsJCAcCBhUKCQgLAgQWAgMBAh4BAheAFiEEGYgtkt2kxADCLA1WzCr0RyFn
+vgMFAmlsPcYFCQ5uMl4ACgkQzCr0RyFnvgN8YQwA4aZVz+ehuL1Kq+fS3kqYt8nf
+LB6f5OX49kL+kUtPBug6xhfog8RinrFq7JmSsZ+id/IZc79OYG3gOxs8HvdIEZe4
+1wL0yfzipjRQuioy9Mh0QFtZFzBQUhbFhRBB3DwJIs2qoJYXuZT03RA5Ld61yzlN
+wzYW+l/Ge8lSTvYdks8RPkY47ZAzbhfxFrzyV6oUObPlJEYMPz3kyq2pKJlh487j
+FKN7f2qeSgTTFz4egVdexJSnQxlCXrXkX91exLKNdp9PaYhODF6Vlypx5n7S0WLJ
+l4MhpBDmdWglaIEOBqXDRV7PaIKnyMBL26TX/67wcwT1PfNBO3S4KE1rfQZDck80
+JvIKq1606TvPxayId860xePbq+I9MHUXzbwduf8a4gR68oriv+N5GOk4CYltEB8r
+1O1iN0LqLtbA3KJ385ipx3X12cbhdobnLEQ3zXMC5BdNjcRKpc+NMWZOnxW4s5p3
+FFraGoIr8hb2yW+ZF82mdACc6Pc1B91nvMnwAYR+wsEUBBMBCgA+AhsDBQsJCAcC
+BhUKCQgLAgQWAgMBAh4BAheAFiEEGYgtkt2kxADCLA1WzCr0RyFnvgMFAmV2RIUF
+CQho4R0ACgkQzCr0RyFnvgM4sgwAo8RkSxARoU3igEm4tkhYBbRFFoz4ssT1RLFo
+/MDfSF735apiXrUKAItyF6YhDCg+C9qSeFqSBESIgtsXlF8aJXwnXz1RdhYBzAyH
+i144JnuwGr3G6pAvka5vvncdyAp6k8hZI/FGKQhsYpf7FEgcHSn19roMKGuJmjcT
+QfRnxl3zUaAWQAhC0AW6iWYdXdbFUe6dNqazSxlJozJdpUpnEwyqLtYNahZWjXAM
+SrqNwBH+J3BiaNorkypLvtUA/n0sxqgS4mun3ObnatFn0Z/uBtZ6mcdMf8JYgkOd
+QuwirNlwBRfvdip/40NntyxRZQ5mHtUqjx1iOzMFEsFG2xz2+7zwYmplY2YpS4aQ
+TPyne6S+M2TmriMOZWdXm3UKLh3GcRAMPPFJ/XTvtDrDyhpcbwoumYjr+0tzNnWY
+EfsLsRrbDdf3O2FvkGiZNbGt3dQNIsTNNSJKjw+OmQDFMo04vutuEy9qC5J99OS3
+Bk87MX4MnH+S7+XDGL7eIOthGBJtwsEUBBMBCgA+AhsDBQsJCAcCBhUKCQgLAgQW
+AgMBAh4BAheAFiEEGYgtkt2kxADCLA1WzCr0RyFnvgMFAmNgUQ4FCQSX9aIACgkQ
+zCr0RyFnvgMjLgwAqH1uSwR/7GWgxf55FMR0ngkv7wQ55LxIe+qGiDLucxpfAFNd
+FqQphTeCq+Ouej3KV6TBpIfvFLSNbaOIZpFE+hO48LMu8Q50+UAQPo3kFMxwcQkB
+GHOwQGWz5D09eSsM+9/OmFFDX2RAermgr0oYXSxv5toF/9bcb90aZNsKLbCkeNeQ
+5Ui8E2Vg4jVRrgiG1VOFSG8+oUzDlHPik69av4z2IGXD4tWWWWocDIb5eQ45jrM2
+TEwpOUFEDCSOzICD9KH0/bncbQUVyczHHTa2cCq4JfZ7CvqzjKvDqtygAUvaE53d
+YNKcVmJx4jjJ3q/4lEh5wCBUHU7QbVyhCs6qjuJXtRGamNY+3Fjajxp6aXuLVDZ4
+ccUQbq2g9IPSl9j7FskuNDABKCA4xv6igAnqrol/+QUJiGCFVVyY02l5s4EBCXvq
+664vWIK1KLqerjlqIGP09SwEOgr6LHLTRF/fg1Go9CwsSHRcdPFGQnbKDX1o6pLD
+nYg1/krEfD8CVFtUwsEUBBMBCgA+FiEEGYgtkt2kxADCLA1WzCr0RyFnvgMFAmEE
+hmgCGwMFCQPCZwAFCwkIBwIGFQoJCAsCBBYCAwECHgECF4AACgkQzCr0RyFnvgNw
+gQv/cwwRUoDCw9C6OjblqEa2eA0OTwkKqT5QcUUtXsMTNHZ5RZ8BHbJjiODTC4uK
+t7IaxkcYH72MLbMQLXAybJJhJ9GsJOvtu7OXS+tKhRjCYrdooFyupjuhq7kzY9Q7
+AsNPYyfGjLzxkJx4DQYe7UzzIsMnFkUBP4nBl4VwzfFyZrKT3WO7+1RWyuNKVwX3
+9vF7JZH8C7j/FE8iFxCHJvfzqs5SMSw4EkvJFRMdqljNkypMEqW4jh3T1RGU3/BX
+fwgJE3MJ+F6kdgXGe9UecKx/Arb1n7JxCfIe4cq/J2jiWnhGXqE/Jay8n4voC0LH
+hrLIjoQQkFMDpqp9rc+bqEjVm/CAWuv+f0F5Ppf4ctGrat+Q0fOTYo80jgFPHw3k
+rJLgAxP9EaIPfX31zFnwyEdwfYy+qaLnmyNAjUR/KAilW9tagXLPXI9TJeJxV5db
+T5XbDY4cx5ksjtuA8fcRyKtcmi4Ujya6rrcrCgr5Ooix8UHUEzDTNZWlMvmG6GMK
+sr3PwnUEEhYKAB0WIQSHEPPenl0uYds4i+4Rmrjw5h2TnAUCZsIz/AAKCRARmrjw
+5h2TnHoZAQCZPPqnhegBvKk1/mWlC7fY2rfnnh9txdvRlAN3WasxdgEAu5my2ys5
+t0lHS6QUYLc/YL8daQi/Z2PMgB4oav/tgwrCwXMEEAEIAB0WIQQbBr9aOCmQ+8ui
+db5Q6Q1VHcFrLgUCZEpd6QAKCRBQ6Q1VHcFrLmdIEACnHbrChcF+s3BNHKm6jLXg
+6E7L7cwrIE3QNAV28ZEwcTtaG4Yxn3EiB2jzm41xlljagu0KdaOaW4DYrFcwNb4Z
+gaSD+UYyTn78t+GQmwQacmoxIslP5r43w9BL6IllGJfSRJuQY3bOq6hLOQBT0i+Q
++IPimiWJwO690ubQdoVN58PpHhnKPQvgZNHcQKH/9qKHxMU0d+e25Z/2sA17LCA0
+Cg6kllOz50B7Zf2NxXWfKjrFUjl2OrCBBac16hzOHwpbg0JbaUUD9bz0rzCN66Gc
+STNiIO46OcUk5loPnUMhfxBA8ehsbPD8Mw7pUaUlgprY/h8OtHVGDRMeqSPyJ/EP
+OOAnodZfnudB8peNWMmgKMOFIL5EfxriBtmkL4HVfjSBkNthMAQeQCd55hmru+Ge
+mrFya/fNqI7XIjNgeDwQjxUFyh1AxjQ+gQDtccO/UHIpy7v8tr8CNvLkO23YVnqZ
+CHfTNlDb6S3cIqattFCaOPCTRgTpj5BOh6G8vqI3g+e7zHWyFDIyAlzmwu4W7Jb+
+ESNkFuZMX4JkB2nLSdvLgbJH8z+rhX29QgWi/raxPLUOvi3GqUGiG2feuF0X34YW
+nsd5Ax3RcGZTBOcsFl6c6nJHHggKdtRi6DPn6kBmZdWjYEttM6gTyeZJL8HiG5/C
+fvM95pbQ75NtcyyFtHd2JsLBcwQQAQoAHRYhBG+3e7icQcjI0tK1mhmyZ4AhAzKf
+BQJhzjhwAAoJEBmyZ4AhAzKfwrIQALMga6m2VIKoPMhBwaN1hLRnrudwx5BfDRaN
+GFcU96NzmmFp7yJ/F1hyjfJeBVNYiFzgOF/qIFgfxfkCH8lvSKNHqq+ne04dWPni
+Ejb7Q4IiRAmj8HKkey1aaKy+TX1Tw/PDNIz1HXi7rpFbGu/VruKzfeh/H+R8AJEi
+3bLnHhYKDEjxMmV6E/P8cpg1GSaxskMRLZIEQaDyE2ySjYfAUbCRiskWpjmho4MV
+rnjYx2JX/c5oBAjrhJoYyjf+PdOeIG3M/Qjtvjdu4WHEAiUAHLGpEeWSw4o10fdj
+gfnM1nrkvrLBWUbOUD5ih0kp5ZyA0GnjpO279+V8QXD8gpghv3M4pSXlFtFHGE7g
+lKu207ZE6IozQjQVujoHaGd03uRpDukhv4B9TWgr9GjdiviBuGT6yWi1SwGV+Efg
+DjB+LjUusfbt75C2syXGL6aC6KWZjTRLnm/06bixLl5Zz9fTSKeo5z0nsM65nfCO
+ncDAVIgumuNmShX4JuCS91cMehjYbaB/Njk0yCXHGTui1b1sB7aFTGvkWEH01oni
+6DiYhuRGtlz+0Q/MgipN1FE1GMntwHrwP8kCmCTXjS7bGwxJxpK+jhQd5kTo2Fa1
+5EIq5BIiZzqP5MZ9bKnCQqxHwaiZEq+8Cblcx9z7uPk6bf2SNgZwyw6EAKFdP1fr
+dm5yPzH8wsFcBBABCgAGBQJhzfBDAAoJEO1sijiDR2RVNVYP/2Mx0ltBnKjY9nNO
+KOmZ7PJTz4Wz11dk4SUbn/bhXg45URqxx81+0UscCXHIGU21eVO6ObdTCGf57zxp
+kDJh18AZufYK6rT1Y9aytGPcWBPHMPWq+gvS38amjuYdtxz9YjeM6tolaZRaFuST
+3gMfVqdeL0pZpsLgJuDoNrl50ioAbbAX2Cs8u5ymlCyCt0nAt1Z9bxnSWXvd+pEY
+hMJMPlLp6SkWRKgcJlRJ8LR4vWVXPGUMT2dS+PlZDakDgYrizW68hu67c3CfNyGr
+1Jtx5G9j9YMbfCQUmbMwR1IdvXl+UBPvakFQsJHGhwi2sRsxlP9vpInKmCXziTtP
+YEp1qsiX6FgI4r60ceWVAOYLb0wl4+bbE6SqlmXsLNKrl+y9FAOqChp/r2KWZdQK
+MognbcZwK98adxbyQCCzYYNTsZVXs+N1orVQ/wLZLFryIzXZhsD2WFtGv86LXQ6T
+9wlTb8MPiR5cIpiHZSyzHamTd9Oi6acK60gx9qcQoybAVXPmBfCr+M4sX0cp0TnR
+iEQVR+CCHe+vr2oDp/SbE+uHSWzH57/tO98oDrdz1frSWlwiB1qdUYFyv3BAjl0h
+O2uzNiRJPt2Zpu8YAMOsz777y89lHwKQyA81xh2ewKe+r4sk7YG2b0wU81uYIrMp
+YwI8wAHHzwxUQbAIpAzQiXJOwmmEwsFzBBABCgAdFiEEfDq5z9IwvTDdAJxZHnCR
+sfFKZKIFAmHN6I8ACgkQHnCRsfFKZKIUBQ/9FdoYLt7ZBw3GkfdomVCBs1K1BFXc
+VPRR7E1+PLTZVlXzkZ78OnCLV7EpharH8PBxyVZnzHBL38+5TKbQePgJujP4A/4z
+DgTlSACPoOT7sRHZ4dFPatVgGKJDW/JO6IvjQ8AazNuD6HjOKRkjBfodHrn5syKG
+TkgMeaiVDn1goZiDEc7vF64Tg4Jl3Bz2enh7d9T0ouu64uDz1/IMI9IO0IE7L0Y0
+rHEYUR7Zz1u0YFUT7RJsdS7SfMyaC/4ck/8u7z9uG+dK0D5KiToMvRoyAJgY76vC
+HEo1SEDCrTNtKrYTCl8KwNCojexcDXslujlh2clAs2pljqVlMxB/eEAcY/N2BEWi
++xjUfUerfwRzzswXyQbQo/BXUc/mTIAPQA9jhRBmKrn7mGx7IWMuEWEC+kwVz1ec
+RM31kxIKoWTnb0i1Mmvl+MbzVuJawhFM+OWIg7DiuEMWTR8oPp3qEdr6o3W9j5dz
+Hl2JhMKmBA0tTuRskhMEvoAfBR7AYM6i/Ynm/breqb0H7iGBhUSMpWbxXlgnx2RO
+k3TMA0lz1WIjBs8IXi83ZTGwyeEVLf7Im0iU+0pkyJ/ex5e1pbOKl16hoOKWwBI0
+Zfj8xnPTyXdAOA1YvCKLrzxEoSgkm/BlwZamPk4YydbNLeTLR7QsHykiYxD2aegI
+8WDslmCj71PCbGzNMVRob21hcyBFLiBEaWNrZXkgKHVzZSBmb3IgZW1haWwpIDxk
+aWNrZXlAaGlzLmNvbT7CwRQEEwEKAD4CGwMFCwkIBwMFFQoJCAsFFgIDAQACHgEC
+F4AWIQQZiC2S3aTEAMIsDVbMKvRHIWe+AwUCaWw9xgUJDm4yXgAKCRDMKvRHIWe+
+A+AeC/wPXLQ3Sx79K6940bpGzLcCHZ0EkL8LEUbvHMmhsJKNR6d7UwiKL3pa2nZh
+lbLfch/q6e/3GiUBQxOSGQQod2KnZJj2yqvF7sFF+0/dQfT1aVgzxfpdwOlDJPJZ
+d2DkZckYB18A0+GbGOcr7jyZK6KRFd00m52CYul94LQdOphmbAw85jQTneirnrTv
+9JAojtaPF9j1y+Yqt8Jr8+bVCiAYjYA1CmRZFac/xz7wlZnFy4F0dONDdr36L84u
+qWqZeyU9dg/PshbNeUlzjVGhfsyj8bx0svawMgKgkgGI84XnGoDMDGLym6AcK9DS
+Myjia7ghXuzRjjozloDDQLVrFk67kkXNJjRKpgDosqLia/Ttm6aMmOp3/7IU6IaN
+3eal12FiU8gHJSenCD5sTmXcE2+nlAKhrX44p7uD6mzDRVauwaoGJyVVgVQf6fTg
+2CcSU6l6b7n9SJ0eQU6lTU5E0QY5ED8WzRxcsa/52Y/JGVSmN2HKveQkEsEpfogr
+PANxbifCwRQEEwEKAD4CGwMFCwkIBwMFFQoJCAsFFgIDAQACHgECF4AWIQQZiC2S
+3aTEAMIsDVbMKvRHIWe+AwUCZXZEiQUJCGjhHQAKCRDMKvRHIWe+A75CC/4yHqbx
+/VXkbu36twuNSN/rWXTdMhTgMWQsH5qVCopv1OPvu7ld6A4RnQFGdISjUS5Xddmy
+k6iLWSgYV5Gpef/1F2X3ppRrndaNy06IHmst7RUgqTuFz7mooJEoGRTMG5RIA1b/
+Sq+i4HcFZ1zbHTAs3LQMoAGf6A5/c4+inLn+8AzO9RqjAkbmPKJcUZaOn7KjkX/o
+uBjX+9BXP3maNMymIR++cGs79IKDRqo1/gG/wUtpL9R+56TI8+hfJCfY4MwwiGyG
+P1Uu4eq9KSpi7L6T15K2dGEfEgCyK5ukTv+tVlUZCx7BtuTZVyuoPMrUYCE2sKrp
+ZT4AlL3v8LeF0PmEVg0Q2vZqM+JrjiAP4EG1lLWWUVaC1roM9vH3zSERRI2pmq6V
+fcXXcOI2b1839V8yAwWWR9FRoklDKZIx4e3G8fZoiZCRyG+VguzTzcubI5vVwf23
+kR61zU+TQPNBc5kNLqgOaymvu87tlRvUAWPk3qB+G/RbfmsQLIQiEOlKWRfCwRQE
+EwEKAD4CGwMFCwkIBwMFFQoJCAsFFgIDAQACHgECF4AWIQQZiC2S3aTEAMIsDVbM
+KvRHIWe+AwUCY2BRCgUJBJf1ogAKCRDMKvRHIWe+A5i/C/9XImMzaVG5J+JDT/ja
+KEM//YBqVjVfpQg7rIFKYK5cNJxSZNYnTNCNsrraebPv7C4OT1OCdjc2adBMXIJE
+/Sky3wygWTkQ56tiH59o9Y7B6z1fFpvgd+Ee5bk7Qyb2qIkSfQzMsUu34EXB6Gze
+kS/N9vbm/6VNPLqCb1cO1njjj+BbM9VCTUBY/qkLJNATrOXzIkfJOAcf177hCDs9
+lyFRd/HEXXlV5nCh4aJ+w41QcqZJXXRLerx9AdlI12xNmU/FnUm/t6hOyv+aqBXf
+EoWV8xYmnX4QcDv/eB6xENBpvZad61w6p3du+586A/vrcsnPk8tnE+jPIf3qa7oD
+h1i9+IpmdONvZPZ7NF4/50o0w8P+sWkMXHhQV5Tpxr+vcLjCIzksZxJS5MRHAdrC
+lt3oF3dgEloXe0XZ5w6g8Sbm7rFMfTpdZ/dr8NA0LDqWjcECHloB1mYSQuq4p8CC
+5Lt0/Yc1PSrPtGYQJDH7M1eCUChEnWLLyLxNxH6+SzijKonCwRQEEwEKAD4WIQQZ
+iC2S3aTEAMIsDVbMKvRHIWe+AwUCYfehswIbAwUJA8JnAAULCQgHAwUVCgkICwUW
+AgMBAAIeAQIXgAAKCRDMKvRHIWe+A4T0C/0frfKN4LCbjimzOBWTVl7h00bmKOXU
+peW44Ii18sqoHgAglZL4vScw9HRwYB7G4zi95VFPcppocy34FYuY7ZrzC0d4qtJy
+ZVEVTodjp9CPlDeOqsSb9nMVELAJ0PineYGCs+6dJCn68Sjll8gEJhc24VBCPWgX
+nVU1/K7EKH62b3152hfrkKd8XK/dEM+BOaSyUS9Rla0avVPsFjo84dGCJRywxcYO
+5O9lYGMs63oIGgIO+BuSOlH0vJAVi98IT+pAXJthCr50gYy9qmmIrYKhcd+GIWab
+Q4/LS8yvWoNWkGuleDVrbeirfVavnQ6jLC4RqHJSjiUF9qkCMWzIvbLytcahhCug
+E7lkh1633FQjdW2bA9K+waQemYvxD3JMaMoArK3DO3XVHMfaXREfc9z56gqvNL5+
+TPsNgaYMxKWT4+J2QkeG1vKcFZDxhUWpYvDmD463nwH33qTz2wnrgPEcNu5hYX+k
+Oeebw55vmgunT6AvrGPeZiUPm/jiQfhLidzCdQQSFgoAHRYhBIcQ896eXS5h2ziL
+7hGauPDmHZOcBQJmwjP5AAoJEBGauPDmHZOch8IA/37eKipVcqODG4pgnqLKsMtb
+0rgUqDP1mjdgnELjFhiOAQDrcg4pDxp/1MQVCvtGe951wQFKdmUqdedyvVgA170z
+A8LBcwQQAQgAHRYhBBsGv1o4KZD7y6J1vlDpDVUdwWsuBQJkSl3uAAoJEFDpDVUd
+wWsuW4YP/0ON19DpoHl6kzxC+A1mu4w7KI5VBaMrUOUYAXxh27DdER2uMTKlJVPP
+46QdK9yAhojO2lohy+/oAGYo8Pagc7hzU1NyretJjTvoJx5XgtXTAoOdihZiSt04
+ng3LGx17tNCJSNyGu8v7whRqdJIoTpjYT/kn2E8UqOIkFp4QFJfrHH7i+sRlyk0B
++MGt6K3vpbhnFbqjm1LpGnS6G+GAXL3xmHWj1BSNCejb2dg9iov58+Sm4jzBXi7M
+UF2kiK43a5qVpp1czk/ztuZsi3G/xX9BQEHFs/GgaLPbvpeZO+HkuQVAIXULteWb
+4dVPFFcytT+Fc8DeDgiFSD1BYQS3YkO8TG3XICTlvms2NS6DnEXDUS/8LLiv4tCy
+WFjZGCiixZewJ6XwSUTTi5t4xF+cqVhTyB/rJbkBcFigk0Gv5ivZs7gbuAYRsTGq
+phSXP7OcCpSqbPaz8rFEkw1db6hH/qsw5cUn17CnzpMxkPVj/bhLd6J02GUPqa24
+tjLPfmaA+WZ+gZBywpxL11Jx39z7bD/S47eMgzPyVR9yns3UttxR7gelzoyBxxAK
+sJP9QhCAgp5DQBVXRFGFJRX3xWSRRPqh7MdstOn/BVGFsMYsc0/xcL6ucEhNh0Nq
+akk8QGh04u7tvVRC4hItgQZ2BlLCKqUcLYu/zjnGC29JUDIsrQpTzsDNBGEEhmgB
+DAC8liudPkt0jbBeUvOvVun2rhPSf7Ge4i4xNO2LjM3o6hy2xsqKqi5QhUl9vasf
+4ObcYh8rh2WxwDWJcLhcQSTRtRAufqjdVtemH9hncUsn28HRuw/E7syYQbRBdXVE
+1BTWFu6byheKcrZ7uV8tpB5FTZSfacLYVUc1GZSy3Q1J04aVqWZWre9fkN9ELVJp
+9sM96s/20Z60xrCyQSW9d+FM/NzB0C7O77QrrTEcotcXGi0GbRD6FCz518lcwiTF
+U+YvhpD9QSekUlujF2keHFMdajH2aHuIzyuu2k8bbN80fG7s/G8C+hNwIlcR05a7
+f0o9loKUwq7E67Fs4P6ZSXmoYU5wGBzsMoM8hM7UsvIDX28KRcz9n753NsIX2HtO
+26sJu3Z9kBj1SFdUTV20sxbKmbF6I0etVpc7x4tc1bBPXEFKeBZph+Zu3mzxcvtl
+ZYraOuqSZFr04Mx+4njIKeHklwr/GW5oaZG1JIPH0HddzPROUpwYXElCJRiEq1Ob
+NesAEQEAAcLA/AQYAQoAJgIbDBYhBBmILZLdpMQAwiwNVswq9EchZ74DBQJpbD33
+BQkObjKPAAoJEMwq9EchZ74DfZ0MAL/S5yrGJp0AZGE+DLFShHNm5rt8M2gLjT/l
+1es8tjvTYdlVpOEi8WL8YnAbuGcbxf60PMrd/OMJL7aEwd3Rrh+WrKs81M2GkGcN
+vJOlSGUTl71wIiqdisKXCrqXkK/4s9BpyOhqvEuW8CyrJtj/pQliFUCAEkc3OntD
+XombMigG2Dj/NLyZj9rOZFNuKq3PtS77CGQl/TBZLt90kCGjWR7U81Mn65fsz4E0
+AqAOEiiu2HH03dKCHlfiLaUlV4GLvFSUnb90EkpLUXsjGcAszavw2Spu6m3Xt2qs
+pUtuYVqdFvHEi0zfyMMc5CGxkgMimpiMWzu2H/ynTkZkBOGPna6MqVvqQTOZn92o
+wpD03Vm0C2KYx2wxE/nZT1m1DKcH49rWyQUBvxYW0G8iVWCE8zlTADZIG1QFSbRE
+7NUYyb6ugV+eAOmjBDw0f2j6V0GzwhXGqLrAqkhG2OYOnwLOO0wa2JCMtPP/9O2C
+HraaQzdF8P+ikKDPyUHRri1fEr1hssLA/AQYAQoAJgIbDBYhBBmILZLdpMQAwiwN
+Vswq9EchZ74DBQJldkSUBQkIaOEsAAoJEMwq9EchZ74DqHwMAO+NYtD6vXL9cxxc
+WG0lP6EtZjdO1VABzGoJr9WRdHUDRNzkVvFTmcroBK8PJKyR0/av/EecVqk6bNQD
+kckmEetevPxxaje6asihviBZV6CRDNaTdilyv2muMQXxG+MSSmFHm/NtKeKBDKQZ
+jHi99i88ILc8fwGWsjQYmBacwBy1scSYvDlkSRfdA2s6Tba5XmoGN83merhMIPwK
+lQFAtSwaXXxZLZLrxtB0KqtAPfZ+ZwHIM8kMKfNUSWH1QVDUM/xcTVPwVkxn0MuF
+SzoXsJjowg2ZAArOoDJ9OBwb9sQL72YPCIeMSn4d0EZquLOg7hoymemOpRr0pT+l
+AFChScILZO/Xt9TK7qoiZrdNnqCRxXBjgmjTLaNwuDLLjBhjnii1QT2J2arJj4z5
+gA6NQMYdwaf2G21grJcJhqyYoeku7xpD3mFR0DytEkT343oGqVrUSLhUizGxddsT
+RHtgYX0sHRYz5o5eKgJARlEI5eBwTfwqLgvn2/sJCBp1QhNv9MLA/AQYAQoAJgIb
+DBYhBBmILZLdpMQAwiwNVswq9EchZ74DBQJjYFExBQkEl/XJAAoJEMwq9EchZ74D
+Y3QL/iRO4ZmH1s8XFRz90KBxcv2DYWFzV9iT7es9NKK6DcUfg3ZnIokdeliBiKPe
+L48oP2hqvCAMsFOJLQ2I4Dmbd1SQQyMWT4cftGl4k/23Bcu2lw9bPJNA7qrJt1Pk
+Bj3c69k5wk4x2F/A21CYJP5kMTJ2vGxCB5jcFxMHbl7aZ9TS/WvHqseeiBzS/fda
+vsJSBbn2bHR78QcHaFU0HKnFjXDrdxypIuozPtgnoCcyrxrthbmwbaQYnl6YgRps
+GnbZfcyrIHnavVnSNXvw6CkqL156gzFIWKNv73eb8f/DUpK8VARZ7f4gMdRfHNbQ
+tyFanEPcz+pNHKN+2pjcw2wl86rJ6ubf8XsQoO+J55Sg9qFN+CCM7otq7M3gPgKO
+iX46kndo+A7q9Qb3mNBDmv8wJPQjHgF6B39i8ZmCqARKX4+wVC61ZVtg35e8jZVs
+AUR8BIf97hv5j2Ps9czQUOSs5QMHo4smZX9570sC8Y2J8SYE6kI5PYeEyYGquExM
+Yu+pwsLA/AQYAQoAJhYhBBmILZLdpMQAwiwNVswq9EchZ74DBQJhBIZoAhsMBQkD
+wmcAAAoJEMwq9EchZ74DamgL/3ipBSBQgvLk/4A7y1mC1fntoaZwwcGJxgXsqCId
+8jo4jBFCyg72nLrDEHYH9AWy1tS4xXdFw0gEAnb2pm87t+GkKoGdeyH2sgFNfMT1
+06OgQRpkVmg7NoVc+qPJcmJCOU1jqSx1PvmJPM/mhTftf+KLljP8a6XhMy1Iuhqo
+nOA1HTtfebo+J/dZmMjCGQKoP2ib8WenI+JAdtuEIw5BUCEhosRnGuMuBbrZwq4+
+usdZQy709l0TeUQLDvKJJrycpy93HrB38Z299QmwlxAgUv3gPjiqf48JC8SqlOnb
+RVcZXNlb0EnypnBxSjo+myC1sysl4sa23XN64+a/yMmDvB2VQykB2bKS1B1aVbGx
+MYtSctV++7kC3BYuFXkTkXWybbzvqPp9Nci2cugeak1qwJRZq7f0s5oJZobaSqkM
+IwXmY9GjsnvWC9WYkWAfYL92TPghI8LOawz6UW2TKePrZlTRlApeOwc5M3K2ep1S
+cW8aJO3XP46cd3aSov0rfNS/zg==
+=p/hP
 -----END PGP PUBLIC KEY BLOCK-----
-- 
2.55.0

